includes/admin/reporting/export/class-batch-export-payments.php

```diff
@@ -52,8 +52,8 @@ public function csv_cols() {
 			'products'     => __( 'Products (Verbose)', 'easy-digital-downloads' ),
 			'products_raw' => __( 'Products (Raw)', 'easy-digital-downloads' ),
 			'skus'         => __( 'SKUs', 'easy-digital-downloads' ),
-			'amount'       => __( 'Amount', 'easy-digital-downloads' ) . ' (' . html_entity_decode( edd_currency_filter( '' ) ) . ')',
-			'tax'          => __( 'Tax', 'easy-digital-downloads' ) . ' (' . html_entity_decode( edd_currency_filter( '' ) ) . ')',
+			'amount'       => __( 'Amount', 'easy-digital-downloads' ),
+			'tax'          => __( 'Tax', 'easy-digital-downloads' ),
```

Comment:
On this export, the currency column already exists, but it's separate from the amount/tax columns. Could it be moved to the same location?

Reply:
Done via https://github.com/easydigitaldownloads/easy-digital-downloads/pull/8676/commits/c510dfaba2bcd9a11672e1001b479fb1d5296049